policy-controller/k8s/index/src/authorization_policy.rs

```diff
@@ -65,6 +66,9 @@ fn target(t: LocalTargetRef) -> Result<Target> {
     if t.targets_kind::<k8s::Namespace>() {
         return Ok(Target::Namespace);
     }
+    if t.targets_kind::<k8s_gateway_api::HttpRoute>() {
```

Comment:
very tiny style nit (sorry): normally, i think we would put a line of whitespace between these
```suggestion
    }
    
    if t.targets_kind::<k8s_gateway_api::HttpRoute>() {
```

Reply:
this wouldn't be consistent with the rest of this function. so if we make ws changes here we should do so above.